client/index.php

```diff
@@ -190,6 +189,13 @@
 try {
     $jiraWebhook->run();
 } catch (\Exception $e) {
+    // For convenience in local development show errors on screen directly

```

Comment:
@lv-dev maybe better to use additional monolog handler in local env?
http://stackoverflow.com/questions/25787258/php-how-to-use-monolog-to-log-to-console-php-out